lib/chrome.js

```
@@ -1301,10 +1242,10 @@ var ChromeObject = function() {
         return this.getEvaluatedValue('(confirm("' + s + '") ? "true" : "false")');
     };
 
-    this.setVendor = function(vendor) {
-        var vendor = vendor.toLowerCase();
+    this.setPublisherName = function(publisherName) {
+        var publisherName = publisherName.toLowerCase();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'publisherName'. Consider to delete it or rename it.

'publisherName' is defined here:


(lint/suspicious/noRedeclare)